packages/manager/modules/overthebox/src/overthebox/actions/index.js

```diff
@@ -0,0 +1,18 @@
+import angular from 'angular';
+
+import 'angular-translate';
+import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+import '@uirouter/angularjs';
+
+import component from './overTheBox-actions.component';
+import routing from './overTheBox-actions.routing';
+
+const moduleName = 'ovhManagerOtbActions';
+
+angular
+  .module(moduleName, [ngTranslateAsyncLoader, 'ui.router', 'ovh-api-services'])
```

Comment:
`ovh-api-services` is injected as a module dependency but not imported.

It is normal?